scripts/data_overlap/light_scenario.py

```diff
@@ -47,3 +47,14 @@ class LightScenario:
 
     instances: List[LightInstance]
     """Instances of this scenario"""
+
+
+@dataclass(frozen=True)
+class ScenarioSpecInstanceIds:
```

Comment:
Is this actually used by scripts? Otherwise we can drop this.

Reply:
Yes in serialization
https://github.com/stanford-crfm/helm/pull/1777/files#diff-76e9c34dbe700181bccf736ef5b18f174929a6c63e8011a74ff2bad6314beafdR1147